xbmc/cores/VideoRenderers/LinuxRendererGLES.cpp

```diff
@@ -607,8 +607,31 @@ void CLinuxRendererGLES::RenderUpdateVideo(bool clear, DWORD flags, DWORD alpha)
     CDVDVideoCodecIMXBuffer *buffer = m_buffers[m_iYV12RenderBuffer].IMXBuffer;
     if (buffer != NULL && buffer->IsValid())
     {
-      ManageDisplay();
-      g_IMXContext.SetBlitRects(m_sourceRect, m_destRect);
+      RENDER_STEREO_MODE stereo_mode = g_graphicsContext.GetStereoMode();
+      if (stereo_mode)
+        g_graphicsContext.SetStereoView(RENDER_STEREO_VIEW_LEFT);
```

Comment:
Why this? It probably explains why the subs are wrong...
If, as @FernetMenta says, this code is only called once, it's unnecessary.
OTOH, the gui code is still called twice and you cannot override the stereo view set by Application.cpp